config/FpConfig.h

```diff
@@ -276,6 +276,11 @@ typedef FwIndexType FwQueueSizeType;
 #define FW_COM_BUFFER_MAX_SIZE 512
 #endif
 
+// Specifies the size of the buffer attached to state machine signals.
+#ifndef FW_SM_SIGNALS_BUFFER_MAX_SIZE
```

Comment:
Shouldn't this be FW_SM_SIGNAL_BUFFER? Instead of SIGNALS?

Reply:
Fixed